roles/infinispan/vars/main.yml

```diff
@@ -21,12 +21,12 @@ jdg:
   home: "{{ jdg_default_installation_path if jdg_enable else infinispan_installation_path }}"
   version: "{{ jdg_version if jdg_enable else infinispan_version }}"
   bundle: "{{ jdg_bundle if jdg_enable else infinispan_bundle }}"
-  installation_path: "{{ jdg_default_installation_path if jdg_enable else infinispan_installation_path }}"
+  installation_path: "{{ (jdg_default_installation_path if jdg_enable else infinispan_installation_path) + ( '-' + jdg_port_offset if jdg_port_offset > 0 else '') }}"
   bind_addr: "{{ jdg_bind_address }}"
-  port: "{{ jdg_port }}"
+  port: "{{ jdg_port + jdg_port_offset }}"
```

Comment:
@guidograzioli I honestly don't know if handling the logic of port_offset here is either brilliant or going to come back to bite us in the butt! (but let's go with it for now)

Reply:
That was only the beginning for this change